index.js

```diff
@@ -1,30 +1,44 @@
 const score=document.querySelector(".score")
-let timeLeft=document.querySelector(".time-left")
+const timeLeft=document.querySelector(".time-left")
 const mole=document.querySelectorAll(".mole")
 const box=document.querySelectorAll(".box")
 
 let hitid=null;
+let totalScore=0;
 
-//remove the presence of class mole
+//initially remove the presence of any class mole 
 function clearMole(){
 mole.forEach((x)=>{
     x.classList.remove("mole")
 })
 }
-//make mole apppear randomly
+
+//make the mole apppear randomly
 function moleAppear(){
   let molelocation=box[Math.floor(Math.random() * 9)];
     molelocation.classList.add("mole")
+    //id of mole location is set to hitid variable
     hitid=molelocation.id
+    //mole is removed after 0.5s which will be the start of new mole appearence
+    setTimeout(() => {
+    molelocation.classList.remove("mole")
+    }, 500);
 }
 
-box.forEach(y=>{
-//     y.addEventListener('mouseup',)
-//     if(y.id=== hitid){
+//mole appears every 0.5seconds by using setInterval
+//  setInterval(moleAppear,500)
 
-//     }
+ //each box is given eventlistener when clicked look for the match of box id that is clicked and id of mole location
+ //when the match is true add score by 10 and display it
+box.forEach(y=>{
```

Comment:
`box` is a singular word but it looks like an iteratable (an object like an array). Unambiguous naming of variables, functions, classes, etc. is really important, although it could seem trivial.

Reply:
So here `boxes` or something similar may be better.